drand/drand.go

```diff
@@ -23,7 +23,8 @@ type DRand struct {
 	bitmap                *bls_cosi.Mask
 	pRand                 *[32]byte
 	rand                  *[32]byte
```

Comment:
are we going to change the number of bytes in future? If so, make it as a variable

Reply:
I will change it to the correct size after adding real vrf.